### PR TITLE
Issue 9058: Port EJB Persistent Timer FAT

### LINKED ISSUE #9058: Port EJB persistent timer core tests to Open Liberty
Port the tests previously in the suite.r6x.base.misc.timer.core package to Open Liberty.

### PR DESCRIPTION
**Pull request by tkburroughs**
Added a core set of EJB persistent timer FAT

Also removed an unnecessary dependency in the com.ibm.ws.couchdb_fat project.

fixes #9058

### CODE REVIEW

**Comment by tkburroughs** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.timer.persistent_fat/test-applications/PersistentTimerCoreEJB.jar/resources/META-INF/ibm-ejb-jar-bnd.xmi [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ejbbnd:EJBJarBinding xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:ejbbnd="ejbbnd.xmi" xmlns:ejb="ejb.xmi" xmi:id="EJBJarBinding_1">
+  <ejbJar href="META-INF/ejb-jar.xml#ejb-jar_ID"/>
+  <ejbBindings xmi:id="EnterpriseBeanBinding_1" jndiName="suite/r6x/base/misc/timer/core/ejb/StatelessHome">
```
Comment (brideck) [inline dev/com.ibm.ws.ejbcontainer.timer.persistent_fat/test-applications/PersistentTimerCoreEJB.jar/resources/META-INF/ibm-ejb-jar-bnd.xmi]: Not currently used, but I would recommend updating the jndiNames so they don't have the old 'suite' naming anymore.

Comment (tkburroughs) [reply dev/com.ibm.ws.ejbcontainer.timer.persistent_fat/test-applications/PersistentTimerCoreEJB.jar/resources/META-INF/ibm-ejb-jar-bnd.xmi]: @brideck Thanks for catching; has now been fixed.

**Comment by tkburroughs** [comment]
#build

**Pull request merged**